man/org.freedesktop.systemd1.xml

```diff
@@ -277,6 +277,7 @@ node /org/freedesktop/systemd1 {
       GetDynamicUsers(out a(us) users);
       DumpUnitFileDescriptorStore(in  s name,
                                   out a(suuutuusu) entries);
+      Describe(out s json);
```

Comment:
hmm, I am pretty sure there also should be a Describe() method on the unit object (and a DesribeUnit() call on the Manager object)

Reply:
can we add a flags param, right from the beginning please, might make sense to suppress what the description includes (i.e. exclude unit descriptions or so)